gct/network.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         self.id = uuid.uuid1().hex
 
     def __repr__(self) -> str:
+        # TODO: Replace with return f"{self.name}({self.line_start}, {self.line_end}, {self.type})"
```

Comment:
graph gets cluttered for long function names. Easier to navigate just by looking at start lineno. Best approach would be to inject JS into SVG s.t. when you click on the node, it'd take you to relevant code block in `/code/HASH` for web version.